[cairo] Unable to install

Issue on `i686`:
```
checking for cairo's OpenGLESv2 surface backend feature... 
checking for glesv2... no
checking GLES2/gl2.h usability... no
checking GLES2/gl2.h presence... no
checking for GLES2/gl2.h... no
checking whether cairo's OpenGLESv2 surface backend feature could be enabled... no (glesv2.pc nor OpenGL ES 2.0 headers not found)
configure: error:  OpenGLESv2 surface backend feature could not be enabled
cairo failed to install: `./configure             --prefix=/usr/local             --libdir=/usr/local/lib             --enable-xlib             --enable-xlib-xcb             --enable-glesv2` exited with 1
```

Issue on `x86_64`:
```
cairo-polygon.c:38:22: fatal error: cairoint.h: No such file or directory
 #include "cairoint.h"
                      ^
compilation terminated.
make[3]: *** [Makefile:2573: cairo-polygon.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: *** [Makefile:2573: cairo-pattern.lo] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/cairo-1.15.10.tar.xz.dir/cairo-1.15.10/src'
make[2]: *** [Makefile:2164: all] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/cairo-1.15.10.tar.xz.dir/cairo-1.15.10/src'
make[1]: *** [Makefile:914: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/cairo-1.15.10.tar.xz.dir/cairo-1.15.10'
make: *** [Makefile:763: all] Error 2
cairo failed to install: `make V=0 -j2` exited with 2
```

--- a/packages/mesa.rb
+++ b/packages/mesa.rb
@@ -3,22 +3,14 @@ require 'package'
 class Mesa < Package
   description 'Open-source implementation of the OpenGL specification'
   homepage 'https://www.mesa3d.org'
-  version '17.3.2'
-  source_url 'https://mesa.freedesktop.org/archive/mesa-17.3.2.tar.xz'
-  source_sha256 'e2844a13f2d6f8f24bee65804a51c42d8dc6ae9c36cff7ee61d0940e796d64c6'
+  version '17.3.6'
+  source_url 'https://mesa.freedesktop.org/archive/mesa-17.3.6.tar.xz'
+  source_sha256 'e5915680d44ac9d05defdec529db7459ac9edd441c9845266eff2e2d3e57fbf8'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-17.3.2-chromeos-x86_64.tar.xz',
 
   })
   binary_sha256 ({
-    aarch64: 'a471dcc099c172ad9113f93adfa5878aac25235a9db60c1d4c68d8c1a7df112f',
-     armv7l: 'a471dcc099c172ad9113f93adfa5878aac25235a9db60c1d4c68d8c1a7df112f',
-       i686: '7a01dc3d49956d87b246a56d9c4ed0f14fc12945d56f1735bccb9f2477c7426e',
-     x86_64: 'f069566c26b2b125c7d86afcd410cc90ca75ea1a62ad747271a6aa2f1f48dbec',
 
   })
 
@@ -46,10 +38,12 @@ class Mesa < Package
             --with-gallium-drivers=nouveau \
             --with-dri-drivers=nouveau \
             --enable-egl \
+            --enable-gles1 \
+            --enable-gles2 \
             --with-platforms=x11,drm,wayland \
             --enable-gbm \
             --enable-xa \
-            --enable-dri"    
+            --enable-dri"
     system "make"
   end
 

